src/test/java/sigma/TaskListTest.java

```diff
@@ -0,0 +1,74 @@
+package sigma;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import sigma.Parser;
+import sigma.TaskList;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import static org.junit.jupiter.api.Assertions.*;
+import sigma.task.Task;
```

Comment:
As the suggestion from the [JavaDocs](https://se-education.org/guides/conventions/java/intermediate.html), I think you can split the imported packages into 3 part, the sigma part, JUnit part and java part